frontend/lib/modules/lend_library/lend_library.dart

```diff
@@ -0,0 +1,335 @@
+          },
+          child: Text('Lend New Item'),
+        ),
+      ),
+      SizedBox(height: 10),
+    ];
+
+    return AppScaffoldComponent(
+      body: ListView(
+        children: [
+          Align(
+            alignment: Alignment.center,
+            child: Container(
+              padding: EdgeInsets.only(top: 20, bottom: 30, left: 10, right: 10),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  ...columnsCreate,
+                  Align(
+                    alignment: Alignment.center,
+                    child: Form(
+                      key: _formKey,
+                      autovalidateMode: AutovalidateMode.onUserInteraction,
+                      child: Wrap(
+                        spacing: 10,
+                        runSpacing: 10,
+                        alignment: WrapAlignment.center,
+                        children: [
+                          SizedBox(
+                            width: 200,
+                            child: _inputFields.inputText(context, _filters, 'title', hint: 'title',
+                              label: 'Filter by Title', debounceChange: 1000, onChange: (String val) {
+                              _getLendLibraryItems();
+                            }),
+                          ),
+                        ]
+                      ),
+                    ),
+                  ),
+                  Align(
+                    alignment: Alignment.center,
+                    child: _buildLendLibraryItemResults(context, currentUserState),
+                  ),
+                ]
+              )
+            )
+          )
+        ]
+      )
+    );
+  }
+
+  void checkFirstLoad() {
+    if (!_firstLoadDone && _locationLoaded) {
+      _firstLoadDone = true;
+      _getLendLibraryItems();
+    }
+  }
+
+  void _init() async {
+    if (!_skipCurrentLocation) {
+      var coordinates = await _location.getLocation();
+      if (coordinates.latitude != null) {
+        _filters['lat'] = coordinates.latitude!;
+        _filters['lng'] = coordinates.longitude!;
+        setState(() {
+          _filters = _filters;
+        });
+      }
+      _locationLoaded = true;
+      checkFirstLoad();
+    }
+  }
+
+  Widget _buildMessage(BuildContext context) {
+    if (_message.length > 0) {
+      return Container(
+        padding: EdgeInsets.only(top: 20, bottom: 20, left: 10, right: 10),
+        child: Text(_message),
+      );
+    }
+    return SizedBox.shrink();
+  }
+
+  _buildLendLibraryItem(LendLibraryItemClass lendLibraryItem, BuildContext context, var currentUserState) {
+    var buttons = [];
+    if (lendLibraryItem.userIdOwner == currentUserState.currentUser.id) {
+      buttons = [
+        ElevatedButton(
+          onPressed: () {
+            Provider.of<LendLibraryItemState>(context, listen: false).setLendLibraryItem(lendLibraryItem);
+            context.go('/lend-library-save');
```

Comment:
I don't have strong opinions on this but I've been using the enum e.g.`Routes.lendLibrary` so I don't have to type the path, do you have a preference?

Reply:
And the benefit of that is if we change the URL we don't have to update in multiple places? Urls may be saved by users so I don't expect them to change, so I'm not sure that use case would happen enough to be worth the extra code to map and import Routes. Unless there is another reason to use a variable reference?

Reply:
True... The other benefit for me is code hinting... so I don't have to type out the whole string, just select the route name 😬

Reply:
Oh okay, well I don't feel super strongly about it so up to you!